src/OnlineSales/Tasks/SyncIpDetailsTask.cs

```diff
@@ -78,6 +76,11 @@ internal override void ExecuteLogTask(List<ChangeLog> nextBatch)
         }
     }
 
+    protected override bool IsTypeInWork(Type type)
+    {
+        return typeof(IHasCreatedByIpAndUserAgent).IsAssignableFrom(type) && typeof(IHasUpdatedByIpAndUserAgent).IsAssignableFrom(type);
```

Comment:
It should be || operant, not &&. It is enough to support just one interface to be included.